core/controllers/admin.py

```diff
@@ -631,3 +632,21 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        if feconf.CAN_SEND_EMAILS:
+            response = {
+                'msg': 'Success! Mail sent to admin.',
+            }
+            dummy_email_name = 'DUMMY_SYSTEM_NAME'
```

Comment:
Do not use dummy names and do not use dummy addresses here. Use feconf.SYSTEM_EMAIL_ADDRESS for the recipient email. Use the current user's username as the sender name.

Reply:
(Alternatively, you can use the system username as the sender name, and in the text of the email, include the current user's username somehow.)

Reply:
Done.